Team18Project/app/src/main/java/com/example/team18project/Player.java

```diff
@@ -131,4 +122,56 @@ public String getUid() {
     public void setUid(String uid) {
         this.uid = uid;
     }
+
+    public void addQRCode(QRCode qrCode) {
+        this.codes.add(qrCode);
+    }
+
+    public void removeQRCode(QRCode qrCode) {
+        assert (this.codes.contains(qrCode)); // throw an exception if the qrCode is not in codes
+        this.codes.remove(qrCode);
+    }
+
+    /**
+     * @return retrieve the sum score of all the QR codes owned by the player
```

Comment:
Javadocs should have a line describing what the method does separate from the @return